components/cdc/src/service.rs

```diff
@@ -32,6 +32,7 @@ const CDC_MSG_MAX_BATCH_SIZE: usize = 128;
 // Assume the average size of event is 1KB.
 // 2 = (CDC_MSG_MAX_BATCH_SIZE * 1KB / CDC_EVENT_MAX_BATCH_SIZE).ceil() + 1 /* reserve for ResolvedTs */;
 const CDC_EVENT_MAX_BATCH_SIZE: usize = 2;
+const CDC_MSG_QUEUE_BOUND: usize = 1024 * 1024;
```

Comment:
Better to make this parameter tunable, otherwise CDC thread would potentially be blocked under heavy stress.